tools/post-upgrade.sh

```diff
@@ -2,7 +2,9 @@
 
 # Performs any necessary steps after the main upgrade process is complete.
 
-[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+# For some reason NVM_DIR isn't set at this point
+export NVM_DIR=/home/pi/.nvm
+\. "$NVM_DIR/nvm.sh"  # This loads nvm
```

Comment:
It's because installing nvm appends those lines to the end of .bashrc and expects you to open a new terminal in order to activate nvm. So if you happen to install nvm into the current shell, you need to execute both of those lines to activate it.

nvm is actually a ton of aliases that need to be sourced into the current shell, rather than being a program.

Reply:
This is running within a process which should have had .bashrc executed already

Reply:
But yes, you're right, both lines are necessary